include/openssl/safestack.h

```diff
@@ -48,6 +48,10 @@ extern "C" {
     { \
         return OPENSSL_sk_reserve((OPENSSL_STACK *)sk, n); \
     } \
+    static ossl_inline int sk_##t1##_reserve_at_least(STACK_OF(t1) *sk, int n) \
```

Comment:
... and here

Reply:
Once the reverse property string lookup was removed, this isn't required anymore.  It was a bit of a kludge so better without it.